model/model_training/tests/test_datasets.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 from model_training.custom_datasets import QA_DATASETS, SUMMARIZATION_DATASETS, get_one_dataset
 from model_training.custom_datasets.dialogue_collator import DialogueDataCollator
-from model_training.utils import get_tokenizer
+from model_training.utils.utils import get_tokenizer
```

Comment:
Personally I don't like utils.utils .. could be solved via __init__.py  in utils dir ... but no reason not to merge this now.